stonesoup/sensormanager/reward.py

```diff
@@ -157,28 +160,32 @@ class ExpectedKLDivergence(RewardFunction):
     the action and resulting measurement from that action.
     """
 
-    predictor: ParticlePredictor = Property(default=None,
-                                            doc="Predictor used to predict the track to a "
-                                                "new state. This reward function is only "
-                                                "compatible with :class:`~.ParticlePredictor` "
-                                                "types.")
-    updater: ParticleUpdater = Property(default=None,
-                                        doc="Updater used to update the track to the new state. "
-                                            "This reward function is only compatible with "
-                                            ":class:`~.ParticleUpdater` types.")
-    method_sum: bool = Property(default=True, doc="Determines method of calculating reward."
-                                                  "Default calculates sum across all targets."
-                                                  "Otherwise calculates mean of all targets.")
+    predictor: Predictor = Property(default=None,
+                                    doc="Predictor used to predict the track to a "
+                                        "new state. This reward function is only "
+                                        "compatible with :class:`~.ParticlePredictor` "
+                                        "types.")
+    updater: Updater = Property(default=None,
+                                doc="Updater used to update the track to the new state. "
+                                    "This reward function is only compatible with "
+                                    ":class:`~.ParticleUpdater` types.")
+    method_sum: bool = Property(default=True,
+                                doc="Determines method of calculating reward."
+                                    "Default calculates sum across all targets."
+                                    "Otherwise calculates mean of all targets.")
+    data_associator: DataAssociator = Property(default=None,
+                                               doc="Data associator for associating "
+                                                   "detections to tracks when "
+                                                   "multiple sensors are managed.")
+
+    return_tracks: bool = Property(default=False,
+                                   doc="A flag for allowing the predicted track, "
+                                       "used to calculate the reward, to be "
+                                       "returned.")
```

Comment:
If the MCTS methods aren't specific to this reward function it might be useful to add this in for the UncertaintyReward at some point too so it can be used more widely (either in this PR or raise an issue and address at a later date)